docs/meshstack.aws.avm-tutorial.md

```diff
@@ -0,0 +1,78 @@
+---
+id: meshstack.aws.avm-tutorial
+title: "Tutorial: Account Vending Machine"
+---
+
+> This tutorial is work in progress!
+
+Purpose: show you how to create secure landing zones for AWS.
+
+Terms:
+
+- **Managed Account** An AWS Account managed under this landing zone.
+- **CFN** shorthand for AWS CloudFormation
+
+## Automation Account
+
+Setup these resources on the Automation Account, here expressed as CloudFormation Templates.
+
+### Acount Vending Machine
+
+Use your existing AVM, we assume it has a single lambda as an Entry Point
```

Comment:
Maybe more like this?

To use your existing AVM, make sure it has a single lambda as its Entry Point